### ISSUE TITLE
🐛 possible raise condition when using `biome lsp-proxy` and having a broken config and formatting a vue file

### ISSUE THREAD

**Issue by AQMpolyface** [biomejs/biome#6220]
### Environment information

```block
CLI:
  Version:                      1.9.4
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           windows

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  NO_COLOR:                     unset
  TERM:                         "xterm-256color"
  JS_RUNTIME_VERSION:           "v23.11.0"
  JS_RUNTIME_NAME:              "node"
  NODE_PACKAGE_MANAGER:         unset

Biome Configuration:
  Error:                        organizeImports has an incorrect type, expected an object, but received a boolean.
  Error:                        Found an unknown key `nursery/useForOf`.
  Error:                        Found an unknown key `typescript/noExplicitAny`.
  Status:                       Loaded with errors
  Formatter disabled:           false
  Linter disabled:              false
  Organize imports disabled:    false
  VCS disabled:                 true

Workspace:
  Open Documents:               0
```

### What happened?

so when using either dprint or biome as formatter in helix, biome lsp-proxy crashes:
`2025-06-04T10:50:07.655 helix_lsp::transport [ERROR] biome-lsp err <- "Biome encountered an unexpected error\n"
2025-06-04T10:50:07.655 helix_lsp::transport [ERROR] biome-lsp err <- "\n"
2025-06-04T10:50:07.655 helix_lsp::transport [ERROR] biome-lsp err <- "This is a bug in Biome, not an error in your code, and we would appreciate it if you could report it to https://github.com/biomejs/biome/issues/ along with the following information to help us fixing the issue:\n"
2025-06-04T10:50:07.655 helix_lsp::transport [ERROR] biome-lsp err <- "\n"
2025-06-04T10:50:07.655 helix_lsp::transport [ERROR] biome-lsp err <- "Source Location: crates\\biome_service\\src\\settings.rs:78:14\n"
2025-06-04T10:50:07.655 helix_lsp::transport [ERROR] biome-lsp err <- "Thread Name: tokio-runtime-worker\n"
2025-06-04T10:50:07.655 helix_lsp::transport [ERROR] biome-lsp err <- "Message: You must have at least one workspace.\n"
2025-06-04T10:50:07.655 helix_lsp::transport [ERROR] biome-lsp err <- "\n"
2025-06-04T10:50:08.099 helix_view::document [ERROR] Formatter error: C:\Users\myname\Desktop\mysqueal\biome.json:3:22 deserialize ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
`
my languages.toml helix config:

```toml
 
[language-server.biome-lsp]
command = "biome"
args = ["lsp-proxy", "--stdio"]

[language-server.tailwindcss-language-server]
command = "tailwindcss-language-server"
args  = ["--stdio"]

[language-server.emmet-lsp]
command = "emmet-ls"
args  = ["--stdio"]

[[language]]
name = "vue"
auto-format = true
# either dprint or biome, doesnt crashes with prettier
# formatter = { command = 'dprint', args = ["fmt", "--stdin", "vue"] }
formatter = { command = "biome", args = ["format","--fix", "--stdin-file-path", "fie.vue"] }

language-servers = ["vuels", "typescript-language-server",  "tailwindcss-language-server", "biome-lsp"] 

[[language-server.typescript-language-server.config.plugins]]
name = "@vue/typescript-plugin"
location = "C:\\Users\\myname\\AppData\\Roaming\\npm\\node_modules\\@vue\\typescript-plugin"
languages = ["vue"]
```

### Expected result

it should either dies gracefully or not crash

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by AQMpolyface** [comment]
also, here is the broken biome.json:
```json
{
  "$schema": "https://biomejs.dev/schemas/1.4.0/schema.json",
  "organizeImports": true,
  "linter": {
    "enabled": true,
    "rules": {
      "recommended": true,
      "nursery/useForOf": "off",
      "typescript/noExplicitAny": "off"
    }
  }
}
```

**Comment by ematipico** [comment]
I recognise the error, and it will be fixed in the upcoming v2. Real soon! In the meantime you can use the beta

**Comment by AQMpolyface** [comment]
oh ok thanks!
